plugins/uniques.js

```diff
@@ -552,7 +522,27 @@ window.plugin.uniques.highlighterDroned = {
 window.plugin.uniques.setupCSS = function() {
   $("<style>")
   .prop("type", "text/css")
-  .html('@include_string:uniques.css@')
```

Comment:
either delete `uniques.css` or use it, using it sounds better

Reply:
While I have no idea where the old code was introduced, the latest commit moves the code back to uniques.css